pandas/core/arrays/arrow/array.py

```diff
@@ -258,9 +258,19 @@ def _from_sequence_of_strings(
 
             scalars = to_datetime(strings, errors="raise").date
         elif pa.types.is_duration(pa_type):
-            from pandas.core.tools.timedeltas import to_timedelta
+            try:
+                # GH51175: test_from_sequence_of_strings_pa_array
+                # attempt to parse as int64 reflecting pyarrow's
+                # duration to string casting behavior
+                if isinstance(strings, (pa.Array, pa.ChunkedArray)):
+                    scalars = strings
+                else:
+                    scalars = pa.array(strings, type=pa.string(), from_pandas=True)
```

Comment:
Can we cast this directly? This might make 2 copies?

Reply:
Do you mean cast from string -> duration directly? If so, no, pyarrow 11 doesn't seem to allow that:

```
import pyarrow as pa

durations = pa.array([1], type=pa.duration("s"))
strings = durations.cast(pa.string())

strings.cast(pa.duration("s")) # -> raises ArrowNotImplementedError
```

Reply:
No I meant:

```
scalars = pa.array(strings, type=pa.string(), from_pandas=True)
scalars = scalars.cast(pa.int64())
```

in the else case. Any reason we are not using int64 directly?

Reply:
sorry, I'm confused with your comment. The `if` block needs to be converted from string -> int64 as well so i'm not following why you would put the cast to int64 inside the `else` block?

```
if isinstance(strings, (pa.Array, pa.ChunkedArray)):
    scalars = strings
else:
    scalars = pa.array(strings, type=pa.string(), from_pandas=True)
scalars = scalars.cast(pa.int64())
```

Reply:
I am talking about doing

```
scalars = pa.array(strings, type=pa.int64(), from_pandas=True)
```
in the else block.
Sorry if that wasn't clear. Seems a bit confusing that we create the array with string before casting to int64

Reply:
oh I see. pyarrow does not allow that:

```
pa.array(["1"], type=pa.int64(), from_pandas=True)

# ArrowInvalid: Could not convert '1' with type str: tried to convert to int64
```

Reply:
Yikes, thx